darli/modeling/functional/state_space.py

```diff
@@ -15,6 +15,9 @@ def __init__(
         else:
             self.__space = CasadiStateSpace(model)
 
+    def set_integrator(self, integrator: Integrator):
+        return self.__space.set_integrator(integrator)
```

Comment:
Better to state that we ask not for instance of Integrator, rather its costructor